src/floating-focus.js

```diff
@@ -188,8 +188,7 @@ export default class FloatingFocus {
 
 	getFloaterPosition(target) {
 		const targetStyle = window.getComputedStyle(target);
-		// default 4 px padding
-		const padding = parseFloat(targetStyle.outlineOffset) || 4;
+		const padding = parseFloat(targetStyle.outlineOffset || '4px');
```

Comment:
Ooooh, that's the bug! Nice one. Maybe just drop the default 4px outline?

Reply:
Maybe add a testcase that an offset of 0 actually provides an offset of 0?

Reply:
I think we do need a fallback for in case the outlineOffset is `null` for some reason (e.g. in the tests). Though usually the computed style should just be `0px` so this fallback should rarely be a thing.

Which is all the more reason to just make it `0px` :)